pkg/aws/ec2/ec2.go

```diff
@@ -536,28 +527,17 @@ func (c *Client) GetSecurityGroups(ctx context.Context) (types.SecurityGroupMap,
 
 // GetInstanceTypes returns all the known EC2 instance types in the configured region
 func (c *Client) GetInstanceTypes(ctx context.Context) ([]ec2_types.InstanceTypeInfo, error) {
-	c.limiter.Limit(ctx, "DescribeInstanceTypes")
-	sinceStart := spanstat.Start()
-	instanceTypeInfos := []ec2_types.InstanceTypeInfo{}
-	output, err := c.ec2Client.DescribeInstanceTypes(ctx, &ec2.DescribeInstanceTypesInput{})
-	c.metricsAPI.ObserveAPICall("DescribeInstanceTypes", deriveStatus(err), sinceStart.Seconds())
-	if err != nil {
-		return instanceTypeInfos, err
-	}
-
-	instanceTypeInfos = append(instanceTypeInfos, output.InstanceTypes...)
-
-	for output.NextToken != nil {
-		describeInstanceTypes := &ec2.DescribeInstanceTypesInput{
-			NextToken: output.NextToken,
-		}
-		output, err = c.ec2Client.DescribeInstanceTypes(ctx, describeInstanceTypes)
+	result := []ec2_types.InstanceTypeInfo{}
```

Comment:
same here for the initialization of the slice.

Reply:
Updated